ui/app/registration/controllers/createPatientController.js

```diff
@@ -10,6 +10,8 @@ angular.module('bahmni.registration')
             $scope.addressHierarchyConfigs = appService.getAppDescriptor().getConfigValue("addressHierarchy");
             $scope.disablePhotoCapture = appService.getAppDescriptor().getConfigValue("disablePhotoCapture");
             $scope.showEnterID = configValueForEnterId === null ? true : configValueForEnterId;
+            $scope.showPreRegistrationOption = appService.getAppDescriptor().getConfigValue('showPreRegistrationOption');
```

Comment:
@kavitha-sundararajan is this still required ?

Reply:
No @binduak , sorry I forgot to remove it. removed it now